umh.docs.umh.app/content/en/docs/getstarted/dataacquisitionmanipulation.md

```diff
@@ -93,9 +93,9 @@ The new data source will now appear in the **Data Sources** section.
 
 ![Data Sources Overview](/images/getstarted/dataAcquisitionManipulation/dataSourcesOverview.png?width=80%)
 
-## Connect MQTT Data Sources
+## Connect MQTT Servers
 
-MQTT data sources can be connected to UMH exclusively through Node-RED.
+MQTT servers can be connected to UMH exclusively through Node-RED.
```

Comment:
it does not need to be exclusively through Node-RED, maybe use a different wording? "There are a lot of options to connect an MQTT server to the UMH. Here we will show the quickest method using Node-RED"